code/modules/client/preferences.dm

```diff
@@ -1708,7 +1708,7 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 						default += " ([config.defaultmap.map_name])"
 					for (var/M in config.maplist)
 						var/datum/map_config/VM = config.maplist[M]
-						if(!VM.votable)
+						if(!VM.votable || (SSmapping.config.map_name == VM.map_name && REALTIMEOFDAY - SSticker.round_start_time >= 24000)) //current map will be excluded from the vote if round has lasted more than 40 minutes. 
```

Comment:
Map rotation excludes the current map entirely and IMO that should also be the case here.

Reply:
Addressed to match review